feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/fee/types/AssetConversionFeePayment.kt

```diff
@@ -81,21 +70,7 @@ internal class AssetConversionFeePayment(
     }
 
     private suspend fun encodableAssetId(): Any {
-        val runtime = chainRegistry.getRuntime(paymentAsset.chainId)
-        val assetsType = paymentAsset.requireStatemine()
-        val index = assetsType.id.asNumberOrThrow()
-
-        val pallet = runtime.metadata.module(assetsType.palletNameOrDefault())
-
-        val multiLocation = MultiLocation(
-            parents = BigInteger.ZERO,
-            interior = Junctions(
-                MultiLocation.Junction.PalletInstance(pallet.index),
-                MultiLocation.Junction.GeneralIndex(index)
-            )
-        )
-
-        return multiLocation.toEncodableInstance()
+        return multiLocationConverter.toMultiLocationOrThrow(paymentAsset).toEncodableInstance()
```

Comment:
To support such foreign assets like MYTH

Reply:
Good idea. We can go even futher and simplify ` encodableNativeAssetId` then by converting `chain.utilityAsset` with converter